tests/test_redshift.py

```diff
@@ -182,6 +182,30 @@ def test_redshift_snapshot(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_redshift_vpc_routing(self):
+        factory = self.replay_flight_data('test_redshift_vpc_routing')
+        client = factory().client('redshift')
+        p = self.load_policy({
+            'name': 'redshift-vpc-routing',
+            'resource': 'redshift',
+            'filters': [
+                {'type': 'value',
+                 'key': 'EnhancedVpcRouting',
+                 'value': True}],
+            'actions': [{'type': 'enable-vpc-routing', 'value': False}]},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+
+        # Ensure that the cluster starts to modify EnhancedVpcRouting value.
+        response = client.describe_clusters(
+            ClusterIdentifier=resources[0]['ClusterIdentifier'])
+        cluster = response['Clusters'][0]
```

Comment:
nitpicking, but technically you need a self.assertEqual(resources[0]['ClusterIdentifier'], clusters['ClusterIdentifier']) else this could be comparing apples and oranges if there are multiple clusters extant.